### DISCUSSION TITLE
what's the best approach to add a robots.txt for crawlers.

### DISCUSSION THREAD

**Discussion by QGoos** [reflex-dev/reflex#2036] [Q&A] [answered]
What would be the best approach to adding a robots.txt for a reflex app?

Tried adding a page that's called robots.txt and just having it be a page that returns
rx.text("User-agent: * Disallow: /")
I feel like I'm approaching this wrong. Is there a way to just serve up a txt file instead? for instance can you add a page that uses a txt file instead of a reflex component?

**Comment by picklelo** [comment, marked as answer]
You should be able to just include a `robots.txt` file within your `assets/` directory and it will show up correctly - any static assets you can just throw in there and they will show up at the same path as the filename.
